main_app/admin.py

```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
-from .models import compute_resource_model,profile_model,create_host_model
+from .models import compute_resource_model,profile_model,create_host_model,operating_system_model
```

Comment:
Put this in ascending order and follow character rule per line (ex 80 char/lines ) 
```
from .models import (
    create_host_model,
    compute_resource_model,
    operating_system_model,
    profile_model
)
```

Reply:
ok we will make those change

Reply:
@ntkathole 
Changes done